main/src/main/scala/sbt/Defaults.scala

```diff
@@ -508,18 +509,18 @@ object Defaults extends BuildCommon {
       (cp, s) =>
         val ans: Seq[Analysis] = cp.flatMap(_.metadata get Keys.analysis) map { case a0: Analysis => a0 }
         val succeeded = TestStatus.read(succeededFile(s.cacheDirectory))
-        val stamps = collection.mutable.Map.empty[File, Long]
+        val stamps = collection.mutable.Map.empty[String, Long]
```

Comment:
Is this change expected? `File` isn't the most impressive of types, but it still beats `String` in my book.